src/utilities/theme/color-adjustments.ts

```diff
@@ -679,4 +680,110 @@ export const colorAdjustments: ColorAdjustments = {
       alpha: 0.15,
     },
   },
+  decorativeOneSurface: {
+    baseColor: 'decorative',
+    light: {
+      hue: 56,
+      saturation: 100,
+      lightness: 84,
+    },
+    dark: {
+      hue: 54,
+      saturation: 98,
+      lightness: 46,
+    },
+  },
+  decorativeOneText: {
+    baseColor: 'decorative',
+    light: {
+      hue: 52,
+      saturation: 100,
+      lightness: 18,
+    },
+    dark: {lightness: 100},
+  },
+  decorativeTwoSurface: {
+    baseColor: 'decorative',
+    light: {
+      hue: 30,
```

Comment:
I think this is fine for now just to get something out there, but what I had in mind is these being relative changes in hue based on the role color.

With this approach, it ignores the base decorative color and fixes the color to Shopify’s palette. To get a relative approach working it would take updates to the typings and color factory.